eslint-example/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "eslint-example",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "dependencies": {},
+    "devDependencies": {
+        "eslint": "^3.9.0",
+        "eslint-plugin-mozilla": "^0.2.3"
```

Comment:
Thanks. I've removed the mozilla plugin and added `"globals": {"chrome": true}` and `env: { "browser": true }`